bin/civihr-install

```diff
@@ -0,0 +1,820 @@
+#!/bin/bash
```

Comment:
This is the text from the link you just shared: 

> Executables should have no extension (strongly preferred) **or a .sh extension**

The extension really doesn't have anything to do if the script is executable or not. I can put a .txt extension to file and it will be executable if give it the right permission.

You don't need to use sh to run a bash command. Even it it has the extension, you can install call it like:
```
$ ./civihr-install.sh
$ /full/path/to/civihr-install.sh
$ civihr-install.sh #if the script can be found in $PATH
```

What I said about the user removing the extension is that the user will have to download this script, right? Most will probably do it with curl. Something like this:

```
curl -O https://raw.githubusercontent.com/civicrm/civihr/PCHR-1666/bin/civihr-install.sh
```

And if they want it without the extension, they can just:

```
curl -o civihr-install https://raw.githubusercontent.com/civicrm/civihr/PCHR-1666/bin/civihr-install.sh
```

On the repository, I think it's better to keep the extension. The reason is that you can just look at the file and know that it is a script without having to open it